src/store/consts/uiUtils.ts

```diff
@@ -58,3 +59,14 @@ export const tickerToAddress = (ticker: string): string => {
 export const addressToTicker = (address: string): string => {
   return reversedAddressTickerMap[address] || address
 }
+
+export const initialXtoY = (tokenXAddress?: string, tokenYAddress?: string) => {
+  if (!tokenXAddress || !tokenYAddress) {
+    return true
+  }
+
+  const isTokeXStablecoin = STABLECOIN_ADDRESSES.some(address => address === tokenXAddress)
+  const isTokenYStablecoin = STABLECOIN_ADDRESSES.some(address => address === tokenYAddress)
+
+  return !(!(isTokeXStablecoin && isTokenYStablecoin) && (isTokeXStablecoin || isTokenYStablecoin))
```

Comment:
gpt said that you can minimize this logical expression to this:  `isTokenXStablecoin == isTokenYStablecoin`
from:
`!(!(isTokeXStablecoin && isTokenYStablecoin) && (isTokeXStablecoin || isTokenYStablecoin))`

Reply:
I simplify this expression to :
`isTokeXStablecoin === isTokenYStablecoin || (!isTokeXStablecoin && !isTokenYStablecoin)`, in this case, GTP was not exactly correct